k8s/scalyr-agent-2.yaml

```diff
@@ -1,92 +1,110 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: DaemonSet
 metadata:
+  annotations:
+    deprecated.daemonset.template.generation: "0"
```

Comment:
What does this do?

Reply:
This was added automatically by the `kubectl` tool that upgraded us up from `extensions/v1beta1`. I don't think we need this.